README.md

```diff
@@ -55,10 +55,37 @@ You can use any IDE or code editing tool for developing on any platform. Use you
 
 ## Troubleshooting
 1. If you are having issues with **_Getting Started_**, please reference [React Native's Documentation](https://reactnative.dev/docs/environment-setup)
-2. If you are running into issues communicating with the API please verify your `.env` file is [set up correctly](#getting-started) for the platform you are trying to run.
+2. If you are running into CORS errors like (in the browser dev console)
+   ```sh
+   Access to fetch at 'https://www.expensify.com/api?command=GetAccountStatus' from origin 'http://localhost:8080' has been blocked by CORS policy
+   ```
+   You probably have a misconfigured `.env` file - remove it (`rm .env`) and try again
 
 **Note:** Expensify engineers that will be testing with the API in your local dev environment please refer to [these additional instructions](https://stackoverflow.com/c/expensify/questions/7699/7700).
 
+## Environment variables
+Creating an `.env` file is not necessary. We advise external contributors against it. It can lead to errors when 
+variables referenced here get updated since your local `.env` file is ignored.
+ 
+- `EXPENSIFY_URL_CASH` - The root URL used for the website
+- `EXPENSIFY_URL_SECURE` - The URL used to hit the Expensify secure API
+- `EXPENSIFY_URL_COM` - The URL used to hit the Expensify API
+- `EXPENSIFY_PARTNER_NAME` - Constant used for the app when authenticating.
+- `EXPENSIFY_PARTNER_PASSWORD` - Another constant used for the app when authenticating. (This is OK to be public) 
+- `PUSHER_APP_KEY` - Key used to authenticate with Pusher.com
+- `SECURE_NGROK_URL` - Secure URL used for `ngrok` when testing
+- `NGROK_URL` - URL used for `ngrok` when testing
+- `USE_NGROK` - Flag to turn `ngrok` testing on or off
+- `USE_WDYR` - Flag to turn [`Why Did You Render`](https://github.com/welldone-software/why-did-you-render) testing on or off
+- `USE_WEB_PROXY` ⚠️used in web/desktop development, it starts a server along the local development server to proxy
+   requests to the backend. External contributors should set this to `true` otherwise they'll have CORS errors.  
+   If you don't want to start the proxy server set this explicitly to `false`
```

Comment:
NAB: What do you think of adding this line with more indentation to make it clearer that it belongs to the `USE_WEB_PROXY` as sub bullet point?
```suggestion
    - If you don't want to start the proxy server set this explicitly to `false`
```

Reply:
I think it's clear it belongs to the `USE_WEB_PROXY` bullet, and making it sub bullet point would make it stand out too much. 
I'm ok with it, I just think you haven't seen the rendered content so want you to confirm if you still want this change
![image](https://user-images.githubusercontent.com/12156624/135289611-be0be3d5-e6ff-48f2-9f6d-0cb3ca911ce3.png)

Reply:
@kidroca I have looked on the rendered content, but this is purely personal/subjective preference, there is not guideline for this :) I am fine with the way how you made it if you prefer it! Just wanted to propose other alternative. Thanks for clarifying!

Reply:
Thank you, I just wasn't sure about it